js/app.js

```diff
@@ -10,39 +10,38 @@ var pike= {
     for (var i = 0; i < timeHours.length; i++) {
       var customerForOneHour =  Math.floor(Math.random() * (this.maxCustomers - this.minCustomers + 1)+ this.minCustomers);
       this.customersEachHour.push(customerForOneHour);
-  }
-  
-},
+    }
+
+  },
 
   totalCookiesSoldToday: [],
-  addedTotalCookies: 0,  
+  addedTotalCookies: 0,
 
   totalSold: function() {
-        for (var i=0; i < timeHours.length; i++) {
-          var totalCookiesSold = this.customersEachHour[i] * this.avgCookies;
-          this.totalCookiesSoldToday.push(Math.floor(totalCookiesSold));
-          this.addedTotalCookies = this.totalCookiesSoldToday[i] + this.addedTotalCookies 
-        }
+    for (var i=0; i < timeHours.length; i++) {
+      var totalCookiesSold = this.customersEachHour[i] * this.avgCookies;
+      this.totalCookiesSoldToday.push(Math.floor(totalCookiesSold));
+      this.addedTotalCookies = this.totalCookiesSoldToday[i] + this.addedTotalCookies;
```

Comment:
You can accomplish this same thing by using `+=` instead. Remember that
```javascript
this.addedTotalCookies = this.addedTotalCookies + this.totalCookiesSoldToday[i];
```
is equivalent to
```javascript
this.addedTotalCookies += this.totalCookiesSoldToday[i];
```